templates/zerver/unsubscribe_confirmation_prompt.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal hide" id="unsubscribe-confirmation" tabindex="-1" role="dialog"
+     aria-labelledby="unsubscribe-confirmation-label" aria-hidden="true">
+  <form id="unsubscribe_confirmation_form" class="form-inline">
+    <div class="modal-header">
+      <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+      <b>{{ _('Confirm') }}</b>
+    </div>
+    <div class="modal-body">
+      <p>{{ _('Are you sure you want to leave') }} <span id="unsub_stream_name"></span>?</p>
+      <p>{{ _('Once you leave this private stream, you cannot join again without invitation.') }}</p>
+      <div>
+        <button class="btn btn-primary" type="submit">{{ _('Yes') }}</button>
+        <button class="btn" data-dismiss="modal" type="button">{{ _('Cancel') }}</button>
```

Comment:
I wonder whether we should use more descriptive button text for this -- one suggestion:

"Yes, I want to leave this stream"
"Cancel, I want to stay subscribed"